x-pack/filebeat/module/salesforce/_meta/config.yml

```diff
@@ -0,0 +1,127 @@
+
+    # Oauth Client Secret
+    # var.client_secret: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+
+    # Oauth Token URL
+    # var.token_url: "https://login.salesforce.com/services/oauth2/token"
+
+    # Oauth User, should include the User mail
+    # var.user: "abc.xyz@mail.com"
+
+    # Oauth password, should include the User password
+    # var.password: "P@$$W0₹D"
+
+    # URL, should include the instance_url
+    # var.url: "https://instance_id.my.salesforce.com"
+      
+  login-rest:
+    enabled: false
+
+    # Oauth Client ID
+    # var.client_id: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+
+    # Oauth Client Secret
+    # var.client_secret: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+
+    # Oauth Token URL
+    # var.token_url: "https://login.salesforce.com/services/oauth2/token"
+
+    # Oauth User, should include the User mail
+    # var.user: "abc.xyz@mail.com"
+
+    # Oauth password, should include the User password
+    # var.password: "P@$$W0₹D"
+
+    # URL, should include the instance_url
+    # var.url: "https://instance_id.my.salesforce.com"
+
+  login-stream:
+    enabled: false
+
+    # Oauth Client ID
+    # var.client_id: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+
+    # Oauth Client Secret
+    # var.client_secret: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+
+    # Oauth Token URL
+    # var.token_url: "https://login.salesforce.com/services/oauth2/token"
+
+    # Oauth User, should include the User mail
+    # var.user: "abc.xyz@mail.com"
+
+    # Oauth password, should include the User password
+    # var.password: "P@$$W0₹D"
+
+    # URL, should include the instance_url
+    # var.url: "https://instance_id.my.salesforce.com"
+
+  logout-rest:
+    enabled: false
+
+    # Oauth Client ID
+    # var.client_id: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+
+    # Oauth Client Secret
+    # var.client_secret: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+
+    # Oauth Token URL
+    # var.token_url: "https://login.salesforce.com/services/oauth2/token"
+
+    # Oauth User, should include the User mail
+    # var.user: "abc.xyz@mail.com"
+
+    # Oauth password, should include the User password
+    # var.password: "P@$$W0₹D"
+
+    # URL, should include the instance_url
+    # var.url: "https://instance_id.my.salesforce.com"
+
+  logout-stream:
+    enabled: false
+
+    # Oauth Client ID
+    # var.client_id: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+
+    # Oauth Client Secret
+    # var.client_secret: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+
+    # Oauth Token URL
+    # var.token_url: "https://login.salesforce.com/services/oauth2/token"
+
+    # Oauth User, should include the User mail
+    # var.user: "abc.xyz@mail.com"
+
+    # Oauth password, should include the User password
+    # var.password: "P@$$W0₹D"
+
+    # URL, should include the instance_url
+    # var.url: "https://instance_id.my.salesforce.com"
+
+  setupaudittrail-rest:

```

Comment:
I'm wondering if we should name it: setup-audit-trail. @yug-crest Did you face any problems with dashes there? I'm just considering which form fits better: setupaudittrail vs setup-audit-trail.

Reply:
Valid point, I thought of using `setupaudittrail` as we are using the convention of `-rest` and `-stream` suffix in all the filesets so as to avoid hyphen inconsistencies. To answer your question, I did not face any issues with the dashes, we can definitely go with `setup-audit-trail-rest`. Another option would be `setup_audit_trail-rest`.
Please let me know if you think I should make any changes. Thanks!

Reply:
Could please ping @akshay-saraswat about the preference?

Reply:
I don't have a preference. `setup-audit-trail-rest` is certainly more readable. My suggestion would be to align ourselves with how Salesforce handles it (i.e. `setupaudittrail`) because customers would be familiar with that. Please let me know your thoughts.

https://developer.salesforce.com/docs/atlas.en-us.object_reference.meta/object_reference/sforce_api_objects_setupaudittrail.htm

Reply:
Sure, `setupaudittrail` might be easier to grep for :) Let's stick to that.